src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperFactsService.cs

```diff
@@ -1,11 +1,11 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using Microsoft.AspNetCore.Razor.Language;
-using Microsoft.AspNetCore.Razor.Language.Legacy;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
```

Comment:
The descriptor provider/binding are from legacy. We already have an issue to move the binding out of legacy and just filed one for the descriptor provider: https://github.com/aspnet/Razor/issues/1209